app/controllers/previews_controller.rb

```diff
@@ -1,22 +1,9 @@
 # frozen_string_literal: true
 
-# app/controllers/previews_controller.rb
 class PreviewsController < ApplicationController
   def create
-    @preview = Preview.spawn_preview_worker(preview_params)
-    render json: @preview
-  end
+    PreviewStartJob.perform_later(params[:preview][:id])
```

Comment:
Not sure to understand why you create a job only to create another one?

Reply:
What i did initially was 

1. Find preview over the network
2. spawn the worker

But finding stalled as the manager was already waiting to hear from this call. Maybe that was just a development issue. Instead of making the worker and manger wait for each other i created this delayed job.